common/azure-common-test/pom.xml

```diff
@@ -32,6 +32,13 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
```

Comment:
Remove this legal line

Reply:
I was trying to do the test-common only in the first round. All client sdk changes are not intended to be here. Will fix this. Thanks for pointing out!